mock_area.tcl

```diff
@@ -1,12 +1,22 @@
-read_db $::env(RESULTS_DIR)/../base/2_floorplan.odb
+proc tee {file content} {
+	puts $content
+	puts $file $content
+}
+
+set out_file [lindex [split $::env(OUTPUTS) ":"] 0]
+
+set RESULTS_DIR $::env(WORK_HOME)/results/$::env(PLATFORM)/$::env(DESIGN_NAME)/$::env(FLOW_VARIANT)
```

Comment:
use $::env(RESULTS_DIR) no?

Reply:
This file in run with `openroad`, not through ORFS Makefile, so not all variables are available.

`RESTULS_DIR` can be defined in this action's env in bazel-orfs, if you prefer this solution

Reply:
I think that would be more robust and leave me with less questions when reading the code,  yes.

Reply:
changed to `$::env(RESULTS_DIR)`